src/core/eme/media_keys_infos_store.ts

```diff
@@ -61,3 +61,7 @@ export default class MediaKeysInfosStore {
     this._state.set(mediaElement, null);
   }
 }
+
+const defaultMediaKeysInfosStore = new MediaKeysInfosStore();
```

Comment:
Why making a class ?
We could expose an object with every funcs to modify the unique `_state` WeakMap.